client/js/config/constant.js

```diff
@@ -1 +1,3 @@
-export const APOLLO_CLIENT_ADDRESS = 'http://157.245.224.214:8000/';
+export const APOLLO_AUTH_ADDRESS = 'http://157.245.224.214:8000/';
```

Comment:
Thank you for renaming this const. Make sure you are renaming all the instances of this const. Your branch is not working after renaming. Please make the changes.

Reply:
ok